### ISSUE TITLE
Invalid ARIA markup in grid resize controls (4.1.2)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux-grids#128]
**Ensuring ARIA roles, states, and properties are valid**
The ARIA attributes including aria-valuemax, aria-valuemax and aria-valuemax attributes are not allowed on the INPUT element.

### Impact
Screen readers would not render the controls properly if the aria roles and properties are not implemented in a valid fashion.

```
<input _ngcontent-c19="" aria-label="Width of column" class="sky-grid-column-input-aria-only ng-untouched ng-pristine ng-valid" role="separator" type="range" aria-controls="sky-grid-1-column-0" aria-valuemin="50" aria-valuemax="9999" aria-valuenow="40" sky-cmp-index="0" min="50" max="9999" step="10"/>
```

### Recommendations
The aria-valuemax, aria-valuemax and aria-valuemax attributes are only inherited into the following roles:
	
- progressbar
- scrollbar
- slider
- spinbutton

[AB#1564787](https://blackbaud.visualstudio.com/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/1564787)

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-MattGregg I'm a little confused on this one.

https://www.w3.org/WAI/PF/aria-1.1/roles#range states that you may use these properties on `range` inputs. (Unless I'm misunderstanding).

The first example section on this MDN page also shows using these properties in the way we are using them (it is oddly on the slider page though not using the slider role though - and I also recognize MDN is not the end all source of truth).

**Comment by Blackbaud-MattGregg** [comment]
@Blackbaud-TrevorBurch Yeah, I'm with you on this one being sloppy or they missed the attributes or something. I tested this in Voiceover on a Mac and it works pretty slick. The only thing that was throwing me a bit is what role its picking up. It reads as a '[slider](https://www.w3.org/TR/wai-aria-1.1/#slider)'. Surprised me b/c I was expecting it to be a role=separator. Apparently type must has a higher precedence or something than role so it was using range's subclass for some reason. I don't think role=separator is doing anything here potentially. 

Also, I think Safari is using slider b/c range is specified not to be used in content https://www.w3.org/TR/wai-aria-1.1/#range See Note.  Update here could be to switch it to use slider instead in case there would be a browser-screen reader combo that doesn't understand range.

**Comment by Blackbaud-TrevorBurch** [comment]
We are going to just make the role `slider` deliberate here.

**Comment by Blackbaud-TrevorBurch** [comment]
In `4.1.0`
